docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -73,6 +73,11 @@ The following table shows the inputs supported by the {agent} in {version}:
 |{n}
 |{y}
 
+|Endpoint security
```

Comment:
Wait this says that Endpoint is not supported on Fleet-managed Elastic Agent?

![image](https://user-images.githubusercontent.com/14206422/133164318-a5aa87a2-c579-4fee-89b4-8b12d2f225f0.png)

Reply:
Thanks I got the order of the columns wrong. I updated it to only supported for Fleet-managed agents.